app/packs/src/apps/mydb/elements/details/wellplates/WellplateDetails.js

```diff
@@ -311,7 +311,7 @@ export default class WellplateDetails extends Component {
           {
             !wellplate.isNew && <CommentSection section="wellplate_list" element={wellplate} />
           }
-          <Well style={{ overflow: 'scroll', height: '100%', 'max-height': 'calc(100vh - 375px)' }}>
+          <Well style={{ overflow: 'scroll', height: '100%', 'maxHeight': 'calc(100vh - 375px)' }}>
```

Comment:
die '' bei maxHeight können dann auch weg, oder?

Reply:
Ja. Ich wollte da nur eine von den nervigen warnings in der console entfernen